### ISSUE TITLE
xamarin / azure : can't login 

### ISSUE THREAD

**Issue by FredericDgrv** [Azure/azure-mobile-services#657]
In an Android app running on a real phone with a 4.4.2 (not emulator), Xamarin.Auth runs successfully to retrieve Access Token from Twitter. The problem is when using these credentials against Azure Mobile Services. The method LoginAsync never comes back.. If I put this code in form of a task and I wait after the task, the task is going long running with no end.

Without authentication, the application was working fine and I could push data to the service.

Client = new MobileServiceClient(applicationURL, applicationKey);
JObject token = new JObject();
token.Add("access_token", "here is the token of the user");
Client.CurrentUser = await Client.LoginAsync(MobileServiceAuthenticationProvider.Twitter, token);

Basically, I tried to follow the guideline of this article but it's not working for me : http://azure.microsoft.com/en-us/documentation/articles/partner-xamarin-mobile-services-how-to-use-client-library/

I use the stable version 1.3.1 of WindowsAzure.MobileServices. I can't install the pre-release version because I also need the package Azure Mobile Services SQliteStore and this package is not available in pre-release.    

any ideas how to turn around that problem ?

**Comment by phvannor** [comment]
Can you confirm if you are using the .NET based back end currently?  If that is the case, they do not yet support a client directed login (passing the access token via logicasync()).

**Comment by FredericDgrv** [comment]
It explains my problem. 
Thanks for your message.
